src/Workspaces/CSharp/Portable/CodeGeneration/OperatorGenerator.cs

```diff
@@ -86,23 +87,37 @@ internal static class OperatorGenerator
 
             var operatorDecl = SyntaxFactory.OperatorDeclaration(
                 attributeLists: AttributeGenerator.GenerateAttributeLists(method.GetAttributes(), options),
-                modifiers: GenerateModifiers(),
+                modifiers: GenerateModifiers(method),
                 returnType: method.ReturnType.GenerateTypeSyntax(),
+                explicitInterfaceSpecifier: GenerateExplicitInterfaceSpecifier(method.ExplicitInterfaceImplementations),
                 operatorKeyword: SyntaxFactory.Token(SyntaxKind.OperatorKeyword),
                 operatorToken: operatorToken,
                 parameterList: ParameterGenerator.GenerateParameterList(method.Parameters, isExplicit: false, options: options),
                 body: hasNoBody ? null : StatementGenerator.GenerateBlock(method),
+                expressionBody: null,
                 semicolonToken: hasNoBody ? SyntaxFactory.Token(SyntaxKind.SemicolonToken) : new SyntaxToken());
 
             operatorDecl = UseExpressionBodyIfDesired(options, operatorDecl, parseOptions);
             return operatorDecl;
         }
 
-        private static SyntaxTokenList GenerateModifiers()
+        private static SyntaxTokenList GenerateModifiers(IMethodSymbol method)
         {
-            return SyntaxFactory.TokenList(
-                SyntaxFactory.Token(SyntaxKind.PublicKeyword),
-                SyntaxFactory.Token(SyntaxKind.StaticKeyword));
+            var tokens = ArrayBuilder<SyntaxToken>.GetInstance();
```

Comment:
```suggestion
            using var tokens = TemporaryArray<SyntaxToken>.Empty;
```

Reply:
TemporaryArray is nice when we expect 4 or less items commonly :)

Reply:
Nice to know about TemporaryArray! Thanks. I applied the change.